### PR TITLE
Remove setup.rb and replace with Ansible

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: playbooks/inventory/vagrant.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```
Comment (ekohl) [inline playbooks/inventory/vagrant.py]: Why revert this change?

File: playbooks/inventory/vagrant.py [hunk 2]
```diff
@@ -17,9 +17,18 @@ def parse_args():
 
 
 def list_running_hosts():
+    hosts = {}
+
+    try:
+        status = subprocess.call(["which","vagrant"])
```
Comment (ekohl) [inline playbooks/inventory/vagrant.py]: if you use ```check_call``` then you don't need to check the exit code since it'll raise an exception.

File: playbooks/inventory/vagrant.py [hunk 3]
```diff
@@ -33,7 +42,14 @@ def list_running_hosts():
 
 def get_host_details(host):
     cmd = "vagrant ssh-config {}".format(host)
+    print(cmd)
```
Comment (ekohl) [inline playbooks/inventory/vagrant.py]: This looks like some debug code.

File: playbooks/inventory/vagrant.py (hunk 1 above)
Comment (ehelms) [inline playbooks/inventory/vagrant.py]: Oops -- I was testing issues with this script and thought that was causing it. I will put that back.

**Comment by ehelms** [comment]
Mind checking https://github.com/theforeman/foreman-infra/pull/273/files before I merge this so I can get the CI system updated shortly after I merge this?

**Comment by ekohl** [comment]
Yay!

**Comment by lpramuk** [comment]
@ehelms in nightly automation we used `./setup.rb`
Can you give me a hint for the replacement.
Ansible? What? How to run it? Where to grab it? ;-)

**Comment by ehelms** [comment]
The replacement is to use Ansible. If you are wanting to spin up your own server, you'd clone Forklift, and run `ansible-playbook playbooks/katello.yml` for example.
